packages/core/src/rules/common/assertions/index.ts

```diff
@@ -39,9 +39,8 @@ export const Assertions = (opts: Record<string, Assertion>) => {
     (opt: unknown) => typeof opt === 'object' && opt !== null
   );
 
-  for (const [index, assertion] of assertions.entries()) {
-    const assertId =
-      (assertion.assertionId && `${assertion.assertionId} assertion`) || `assertion #${index + 1}`;
+  for (const [_, assertion] of assertions.entries()) {
+    const assertId = assertion.assertionId;
```

Comment:
Do we need this const at all? Maybe we can simply use `assertion.assertionId` in place of it?